spec/controllers/flood_risk_engine/enrollments/steps_controller/partnership_details_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+module FloodRiskEngine
+  module Enrollments
+
+    describe StepsController do
```

Comment:
As the same mechanism is used to provide the back step at each step, you shouldn't need to test it on every step.